pkgs/games/frozen-bubble/default.nix

```diff
@@ -17,6 +28,11 @@ perlPackages.buildPerlModule {
 
   perlPreHook = "export LD=$CC";
 
+  postInstall = ''
+    mkdir -p $out/share/applications
+    cp ${desktopItem}/share/applications/* $out/share/applications/
```

Comment:
`install -Dm444 -$ out/share/applications ${desktopItem}/share/applications/*`
And then you can drop `mkdir`.

Reply:
Ah yes, `install` -- done, thanks!